axelrod/__init__.py

```diff
@@ -1,24 +1,28 @@
 DEFAULT_TURNS = 200
 
 # The order of imports matters!
+from axelrod.ipd import graph
+from axelrod.ipd.action import Action
+from axelrod.ipd.random_ import random_choice, random_flip, seed, Pdf
+from axelrod.ipd import eigen
+from axelrod.ipd.plot import Plot
+from axelrod.ipd.history import History, LimitedHistory
+from axelrod.player import BasePlayer
+from axelrod.ipd.player import IpdPlayer
+from axelrod.ipd.classifier import Classifiers
+from axelrod.ipd.evolvable_player import EvolvablePlayer
+from axelrod.game import BaseGame
+from axelrod.ipd.game import IpdGame, DefaultGame
+from axelrod.ipd.moran import MoranProcess, ApproximateMoranProcess
+from axelrod.ipd.strategies import *
+from axelrod.ipd.match_generator import *
+from axelrod.ipd.tournament import IpdTournament
```

Comment:
Would the idea be (for now from the pov of keeping the current API) to change this to:

```suggestion
import axelrod.ipd.tournament.IpdTournament as Tournament
```

Or perhaps it would be nicer to have:

```
axelrod.ipd.tournament.IpdTournament
``

as 

```
axelrod.ipd.tournament.Tournament
```

As the namespace takes care of the tournament? So theoretically we would also have:

```
axelrod.<other_game>.tournament.Tournament
```

At some point?

Reply:
And then similarly for the `IpdMatch`? Perhaps just have `axelrod.ipd.match.Match`? Which makes things easier now but I believe also later as when we have multiple game types a `Match` is always the name of the object needed and the namespace takes care of behaviour?

Reply:
I'm not sure we need a new Match for each game -- maybe a generic Match class could work? Same for Tournaments -- maybe these classes don't need to know about the Game and the Players, they just need to be handed Players and a compatible Game.

Reply:
That would be great!